PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -710,6 +710,40 @@ function Start-CopyJob(
     }
 }
 
+#
+# Utility wrapper for invoking commands by opening sessions 
+# for each of the cluster nodes and preserving the session 
+# to be deleted after use.
+#
+
+function New-InvokeCommand (
+    [string[]] $ClusterNodes = @(),
+    [string] $JobName,
+    [scriptblock] $ScriptBlock
+    )
+{
+	$Jobs       = @()
+	$Sessions   = @()
+	$SessionIds = @()
+
+	if ($ClusterNodes.Count -ge 1) {
+					$Sessions = New-PSSession -ComputerName $ClusterNodes

```

Comment:
Check your spaces v. tabs editor setting?

Reply:
Sure. Will fix it.